libraries/AP_HAL_ChibiOS/hwdef/FlywooF745/hwdef-bl.dat

```diff
@@ -0,0 +1,44 @@
+# hw definition file for processing by chibios_pins.py
+# for Flywoo Goku GN F745.
+
+# MCU class and specific type
+MCU STM32F7xx STM32F745xx
+
+# board ID for firmware load (borrowing from Beast F7 for testing)
```

Comment:
Comment needs changing

Reply:
Missed This one. Fixing/committing change now. Thanks